code/core/src/components/components/TourGuide/TourTooltip.tsx

```diff
@@ -115,13 +130,7 @@ export const Tooltip: FC<TooltipProps> = ({
       }
     `;
     document.head.appendChild(style);
-    return () => {
-      const styleElement = document.querySelector('#sb-onboarding-arrow-style');
-
-      if (styleElement) {
-        styleElement.remove();
-      }
-    };
+    return () => document.getElementById(ONBOARDING_ARROW_STYLE_ID)?.remove();
```

Comment:
All this  code looks rather odd to me, considering we're using react, and this is in the manager, and we should have full control. Also considering the Tooltip code got removed in @Sidnioulz's work for a11y.